tests/unit/nupic/research/spatial_pooler_boost_test.py

```diff
@@ -207,7 +205,7 @@ def verifySDRProperties(self):
   def boostTestPhase1(self):
     
     y = numpy.zeros(self.columnDimensions, dtype = uintType)
-
+    self.sp._maxBoost = 1
```

Comment:
Why do you need to set maxBoost to 1? In an end to end test like this it would be better to keep maxBoost fixed throughout since that is how it is normally run. You can update the test to reflect the desired behavior.